opensrp-chw/src/kk/resources/anc_hv_clinic_attendance.properties

```diff
@@ -36,6 +36,7 @@ anc_hv_clinic_attendance.step1.last_clinic_visit.value[3] = More than a month ag
 anc_hv_clinic_attendance.step1.early_clinic_attendance_advice_toaster.text = Advise the pregnant woman to attend clinic as soon as possible.
 
 anc_hv_clinic_attendance.step1.services_received.label = What services did you received as part of your ANC clinic visit
+anc_hv_clinic_attendance.step1.services_received.label_info_text= NOTE: Ask the pregnant woman for her clinic card and discuss the services she has already received. Select the services she has already received from the list provided.
```

Comment:
@maulidkategile - what about the swahili translation for this one?

Reply:
Okay, I am working on it